### DISCUSSION TITLE
Does this block all traffic to the range of udp ports?

### DISCUSSION THREAD

**Discussion by Lockbox313** [Speyedr/socialclub-notification-blocker#7] [Q&A] [answered]
Does this block all traffic including game communications and cloud saves to the range of UDP ports?

**Comment by Speyedr** [comment, marked as answer]
This does not block other game communications such as cloud saving. It only blocks the notifications feature of the SocialClub Overlay.

**Reply by Lockbox313** [reply]
Cheers
